libs/launcher/base_launcher.cpp

```
@@ -400,13 +426,14 @@ ErrorCode BaseLauncher::launch() {
     }
 
     const CameraId default_camera = 0;
-    if (slam_images.find(default_camera) != slam_images.end() && curr_meta.find(default_camera) != curr_meta.end() &&
-        FLAGS_slam_load_and_localize_on_frame == curr_meta.at(default_camera).frame_number) {
+    if (default_camera < slam_images.size() && slam_images[default_camera] != nullptr &&
```

Clang (14.0.6)

[note] +2, including nesting penalty of 1, nesting level increased to 2

(clang)